src/screens/Airdrop/RequirementsScreen.tsx

```diff
@@ -0,0 +1,147 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { Image, Text, StyleSheet, View, TouchableOpacity } from 'react-native';
+import { connect } from 'react-redux';
+
+import { icons } from 'app/assets';
+import { ScreenTemplate, Button, Header } from 'app/components';
+import { Route } from 'app/consts';
+import { ApplicationState } from 'app/state';
+import { actions } from 'app/state/airdrop';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface ActionProps {
+  completeThankYouFlow: Function;
+}
+
+type Props = {
+  navigation: StackNavigationProp<any, Route.AirdropRequirements>;
+} & ActionProps;
+
+class AirdropRequirementsScreen extends Component<Props, {}> {
+  onSoundsGreatPress = () => {
+    this.props.completeThankYouFlow();
+    // this.props.navigation.navigate(Route.Dashboard); WIP
+  };
+
+  onTermsConditionsPress = () => {
+    // this.props.navigation.navigate(Route.AirdropTermsAndConditions); WIP
+  };
+
+  getRequirementsListItems = () =>
+    Object.values(i18n.airdrop.requirements.points).map((requirement, i) => (
+      <View key={i} style={styles.listItem}>
+        <Image source={icons.roundTick} style={styles.image} />
+        <Text style={styles.listText}>{requirement}</Text>
+      </View>
+    ));
+
+  render() {
+    return (
+      <ScreenTemplate
+        header={<Header isBackArrow title={i18n.airdrop.title} />}
+        footer={
+          <>
+            <Button
+              testID="sounds-great"
+              onPress={this.onSoundsGreatPress}
+              style={styles.soundsGreatButton}
+              title={i18n.airdrop.requirements.soundsGreat}
+            />
+            <View style={styles.termsAndConditions}>
+              <Text style={styles.description}>{i18n.airdrop.requirements.termsAndConditions.read}</Text>
+              <TouchableOpacity onPress={this.onTermsConditionsPress}>
+                <Text style={styles.termsText}>{i18n.airdrop.requirements.termsAndConditions.termsAndConditions}</Text>
+              </TouchableOpacity>
+            </View>
+          </>
+        }
+      >
+        <Text style={styles.subtitle}>{i18n.airdrop.requirements.subtitle}</Text>
+        <Text style={styles.description}>{i18n.airdrop.requirements.description}</Text>
+        <View>
+          <Text style={styles.listHeader}>{i18n.airdrop.requirements.listHeader}</Text>
+          {this.getRequirementsListItems()}
+        </View>
+        <View style={styles.dateInfoContainer}>
+          <Text style={styles.description}>{i18n.airdrop.requirements.dateHeader}</Text>
+          <Text style={styles.date}>19/12/2020 1 P.M GMT +1</Text>
```

Comment:
I'm guessing it shouldn't be hardcoded as well

Reply:
it will be added in my PR